.travis.yml

```diff
@@ -1,10 +1,26 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "4.4.2"
+  - '0.10'
+  - 4.4.2
```

Comment:
I think they might be, going to test. Some of these quotes were added by the [lib I used](https://github.com/paolobueno/yaml-merge) to add some of the changes to the 20+ yaml files.

I think it added the quotes around `0.10` so it won't be interpreted as a number, but since `4.4.2` has two dots it's always a string.

On that note I've seen some different versions mainly of 4.x used across the different repos, should we be bothering on testing on a specific version instead of 4 latest or 4.4?